### PR TITLE
Updated DDL, DML

### PR DESCRIPTION
**Pull request by Runi-VN**
(DDL) Add NOT NULL DEFAULT 0 to roof tiles, shed walls & shed floor.

(DML) Add wooden planks (floor), regular boards (more widths), tagpap & accessories.

Added new wooden floor planks for shed
Added more options for regular boards. We now have width 100, 150, 200 & several lengths too.
Added tagpap & tagpap-accessories

### CODE REVIEW

File: extraFiles/Database docs/create(DDL).sql [hunk 1]
```diff
@@ -63,11 +63,11 @@ CREATE TABLE `carportdb`.`orders` (
...
+  `roof_tiles_id` INT NOT NULL DEFAULT 0,
   `shed_width` INT NULL,
   `shed_length` INT NULL,
-  `shed_walls_id` INT NULL,
-  `shed_floor_id` INT NULL,
+  `shed_walls_id` INT NOT NULL DEFAULT 0,
+  `shed_floor_id` INT NOT NULL DEFAULT 0,
   `customer_id` INT NOT NULL,
   `employee_id` INT NOT NULL,
   PRIMARY KEY (`id_order`),
```
Comment (Runi-VN) [inline extraFiles/Database docs/create(DDL).sql]: Added NOT NULL DEFAULT 0 to 
roof_tiles
shed_walls
shed_floor

File: extraFiles/Database docs/insertion(DML).sql [hunk 2]
```diff
@@ -99,21 +99,21 @@ category defines what part of the carport the material belongs to.
 Taking use of default values (auto_increment) for >id_material<. */
```
Comment (Runi-VN) [inline extraFiles/Database docs/insertion(DML).sql]: Fix cm to mm conversion

File: extraFiles/Database docs/insertion(DML).sql [hunk 3]
```diff
@@ -173,3 +173,45 @@ INSERT INTO `carportdb`.`bills` (`retail_price`, `final_price`, `id_order`) VALU
 INSERT INTO `carportdb`.`bills` (`retail_price`, `final_price`, `id_order`) VALUES ('21000', '25995.95', '2');
 INSERT INTO `carportdb`.`bills` (`retail_price`, `final_price`, `id_order`) VALUES ('7800', '9999.95', '3');
 INSERT INTO `carportdb`.`bills` (`retail_price`, `final_price`, `id_order`) VALUES ('13273.44', '11849.99', '4');
+
```
Comment (Runi-VN) [inline extraFiles/Database docs/insertion(DML).sql]: add @Castau's dimensions for roof tiles & add new dimensions for plastic roof.

also:
Added new wooden floor planks for shed
Added more options for regular boards. We now have width 100, 150, 200 & several lengths too.
Added tagpap & tagpap-accessories

File: extraFiles/Database docs/insertion(DML).sql [hunk 4]
```diff
@@ -173,3 +173,45 @@ INSERT INTO `carportdb`.`bills` (`retail_price`, `final_price`, `id_order`) VALU
...
+INSERT INTO `carportdb`.`materials` (`description`, `width`, `length`, `cost_price`, `unit`, `id_category`) VALUES ('Tagpap', '100', '3600', '62.5', 'Stk', '3');
+INSERT INTO `carportdb`.`materials` (`description`, `height`, `width`, `length`, `cost_price`, `unit`, `id_category`) VALUES ('Træplader (Til tagpap)', '0', '0', '0', '1500', 'Pakke', '3');
+
+/* Wooden floor planks */
+INSERT INTO `carportdb`.`materials` (`description`, `height`, `width`, `length`, `cost_price`, `unit`, `id_category`) VALUES ('30x200mm alm. planke (Eg)', '30', '200', '3600', '55', 'Stk', '1');
+INSERT INTO `carportdb`.`materials` (`description`, `height`, `width`, `length`, `cost_price`, `unit`, `id_category`) VALUES ('30x250mm alm. planke (Eg)', '30', '250', '4800', '75', 'Stk', '1');
+/* More options for beklædning & gavl */
+INSERT INTO `carportdb`.`materials` (`description`, `height`, `width`, `length`, `cost_price`, `unit`, `id_category`) VALUES ('25x200mm alm. Bræt', '25', '200', '2400', '35', 'Stk', '3');
+INSERT INTO `carportdb`.`materials` (`description`, `height`, `width`, `length`, `cost_price`, `unit`, `id_category`) VALUES ('25x200mm alm. Bræt', '25', '200', '3200', '47.5', 'Stk', '3');
```
Comment (MalteMagnussen) [inline extraFiles/Database docs/insertion(DML).sql]: Thank you @Runi-VN

**Comment by Runi-VN** [comment]
Applied to DB.

**Pull request closed without merging**